examples/layout-component/components/layout.js

```diff
@@ -1,36 +1,15 @@
-import Link from 'next/link'
 import Head from 'next/head'
+import styles from './layout.module.css'
 
-export default function Layout({
-  children,
-  title = 'This is the default title',
-}) {
+export default function Layout({ children }) {
   return (
-    <div>
+    <>
       <Head>
-        <title>{title}</title>
+        <title>Layouts Example</title>
         <meta charSet="utf-8" />
         <meta name="viewport" content="initial-scale=1.0, width=device-width" />
```

Comment:
```suggestion
```
Should we remove these as they are added by Next.js by default?

Reply:
Yes!